homeassistant/components/nanoleaf/__init__.py

```diff
@@ -9,6 +9,8 @@
 
 from .const import DOMAIN
 
+PLATFORMS = ["light", "button"]
```

Comment:
```suggestion
PLATFORMS = ["button", "light"]
```

Usually we sort these lexicographically